### PR TITLE
Feature/iesp 154

### PR DESCRIPTION
**Pull request by melissa-dellorco**

### CODE REVIEW

**Review by nttdata-rtorsoli — DISMISSED** [review]

File: src/test/java/it/pagopa/interop/probing/caller/unit/consumer/PollingReceiverTest.java [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    eserviceContentDto = EserviceContentDto.builder().eserviceRecordId(1L)
+        .technology(EserviceTechnology.REST).basePath(basePath).build();
+    telemetryDto = TelemetryDto.builder().eserviceRecordId(1L).status(EserviceStatus.OK)
+        .responseTime(12345L).koReason(null).checkTime("12345").build();
+    pollingDto = PollingDto.builder().eserviceRecordId(1L)
+        .responseReceived(OffsetDateTime.now(ZoneOffset.UTC)).status(EserviceStatus.OK).build();
+  }
+
+  @Test
+  @DisplayName("The receiverMessage method is tested.")
```
Comment (galales) [inline src/test/java/it/pagopa/interop/probing/caller/unit/consumer/PollingReceiverTest.java]: please in the label specify which is the use case

File: src/test/java/it/pagopa/interop/probing/caller/unit/dto/EserviceContentDtoTest.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  @BeforeEach
+  void setup() throws IOException {
+    String[] basePath = {"basePath1", "basePath2"};
+    eserviceContentDto = EserviceContentDto.builder().eserviceRecordId(1L)
+        .technology(EserviceTechnology.REST).basePath(basePath).build();
+  }
+
+
+  @Test
+  @DisplayName("Test the utility toString of lombok.")
```
Comment (galales) [inline src/test/java/it/pagopa/interop/probing/caller/unit/dto/EserviceContentDtoTest.java]: which is the purpose of this test?
do we need to verify the output of the `toString` for any business logic?

File: src/test/java/it/pagopa/interop/probing/caller/unit/producer/PollingResultSendTest.java [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+  @BeforeEach
+  void setup() {
+    ReflectionTestUtils.setField(pollingResultSend, "sqsUrl", TEST_URL);
+    pollingDto =
+        PollingDto.builder().eserviceRecordId(1L).responseReceived(OffsetDateTime.now()).build();
+  }
+
+  @Test
+  @DisplayName("The sendMessage method of PollingResultSend class is tested.")
```
Comment (galales) [inline src/test/java/it/pagopa/interop/probing/caller/unit/producer/PollingResultSendTest.java]: same, please define the use case tested

File: src/test/java/it/pagopa/interop/probing/caller/unit/producer/TelemetryResultSendTest.java [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+  @BeforeEach
+  void setup() {
+    ReflectionTestUtils.setField(telemetryResultSend, "sqsUrl", TEST_URL);
+    telemetryDto = TelemetryDto.builder().eserviceRecordId(1L).status(EserviceStatus.OK)
+        .responseTime(12345L).koReason(null).checkTime("12345").build();
+  }
+
+
+  @Test
+  @DisplayName("The sendMessage method of PollingResultSend class is tested.")
```
Comment (galales) [inline src/test/java/it/pagopa/interop/probing/caller/unit/producer/TelemetryResultSendTest.java]: same

File: src/test/java/it/pagopa/interop/probing/caller/unit/util/ClientUtilTest.java [hunk 5]
```diff
@@ -0,0 +1,163 @@
...
+    eserviceContentSoapDto = EserviceContentDto.builder().eserviceRecordId(1L)
+        .technology(EserviceTechnology.SOAP).basePath(basePath).build();
+    telemetryDto = TelemetryDto.builder().eserviceRecordId(1L).status(EserviceStatus.OK)
+        .responseTime(12345L).koReason(null).checkTime("12345").build();
+    telemetryDtoKO = TelemetryDto.builder().eserviceRecordId(1L).status(EserviceStatus.KO)
+        .koReason("KO").checkTime("12345").build();
+  }
+
+  @Test
+  @DisplayName("Test call probing - rest")
```
Comment (galales) [inline src/test/java/it/pagopa/interop/probing/caller/unit/util/ClientUtilTest.java]: shouldn't be there a `Test OK` and `Test KO` also for REST?

File: src/test/java/it/pagopa/interop/probing/caller/unit/consumer/PollingReceiverTest.java (hunk 1 above)
Comment (melissa-dellorco) [reply src/test/java/it/pagopa/interop/probing/caller/unit/consumer/PollingReceiverTest.java]: done

File: src/test/java/it/pagopa/interop/probing/caller/unit/dto/EserviceContentDtoTest.java (hunk 2 above)
Comment (melissa-dellorco) [reply src/test/java/it/pagopa/interop/probing/caller/unit/dto/EserviceContentDtoTest.java]: No, I deleted it

File: src/test/java/it/pagopa/interop/probing/caller/unit/producer/PollingResultSendTest.java (hunk 3 above)
Comment (melissa-dellorco) [reply src/test/java/it/pagopa/interop/probing/caller/unit/producer/PollingResultSendTest.java]: done

File: src/test/java/it/pagopa/interop/probing/caller/unit/producer/TelemetryResultSendTest.java (hunk 4 above)
Comment (melissa-dellorco) [reply src/test/java/it/pagopa/interop/probing/caller/unit/producer/TelemetryResultSendTest.java]: done

File: src/test/java/it/pagopa/interop/probing/caller/unit/util/ClientUtilTest.java (hunk 5 above)
Comment (melissa-dellorco) [reply src/test/java/it/pagopa/interop/probing/caller/unit/util/ClientUtilTest.java]: done

**Review by nttdata-rtorsoli — APPROVED** [review]

**Review by galales — APPROVED** [review]

**Pull request merged**